app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -331,9 +328,8 @@ class HomeFragment : Fragment() {
         super.onViewCreated(view, savedInstanceState)
 
         FragmentPreDrawManager(this).execute {
-            val homeViewModel: HomeScreenViewModel by activityViewModels {
-                ViewModelProvider.NewInstanceFactory() // this is a workaround for #4652
-            }
+            val homeViewModel: HomeScreenViewModel by activityViewModels()
```

Comment:
Here I was wondering why we're not just using the `homeViewModel` field (L115) instead of "looking it up again". Do you remember?